### PR TITLE
Add Tarek to Solang Playground contributers

### PR DESCRIPTION
**Pull request by salaheldinsoliman**
@tareknaser Has been a involved with the Solang ecosystem, in a Hyperledger mentorship, and will be a main contributer to Solang Playground.

### CODE REVIEW

**Comment by salaheldinsoliman** [comment]
I have read and hereby sign the Contributor License Agreement.

**Review by semuelle — APPROVED** [review]

File: applications/Solang_Playground.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Solang Playground
 
 
-- **Team Name:** Salaheldin Soliman
-- **Payment Address:** 0x9036b20920ddf473e796d24afd0fe04d4dde0b68
+- **Team Name:** Salaheldin Soliman and Tarek Elsayed
+- **Payment Address:**
```
Comment (PieWol) [inline applications/Solang_Playground.md:5]: You added another payment address. Does this mean you both want to receive 50% of the total amount?

**Review by PieWol — CHANGES_REQUESTED** [review]

**Review by Noc2 — APPROVED** [review]

File: applications/Solang_Playground.md (hunk 1 above)
Comment (salaheldinsoliman) [reply applications/Solang_Playground.md:5]: Nope, but after each milestone, we will file two invoices (one per each contributer) based on our agreement.
If the exact split is to be amended to this file that should be better of course

Comment (PieWol) [reply applications/Solang_Playground.md:5]: Got it, thanks for clarifying.

**Pull request merged**